compiler/src/dotty/tools/dotc/config/Feature.scala

```diff
@@ -88,8 +88,7 @@ object Feature:
   /** If current source migrates to `version`, issue given warning message
    *  and return `true`, otherwise return `false`.
    */
-  def warnOnMigration(msg: Message, pos: SrcPos,
-      version: SourceVersion)(using Context): Boolean =
+  def warnOnMigration(msg: Message, pos: SrcPos, version: SourceVersion)(using Context): Boolean =
     if sourceVersion.isMigrating && sourceVersion.stable == version
        || (version == `3.0` || version == `3.1` || version == `3.2`) && migrateTo3
```

Comment:
`3.2` is no longer considered equivalent to `3.0`
```suggestion
       || (version == `3.0` || version == `3.1`) && migrateTo3
```

Reply:
this is partly the reason I wanted to introduce a "series" concept in https://github.com/lampepfl/dotty/pull/14629, so you don't need to remember all the parts of code where versions are considered equivalent